docs/source/platform/operation-registry.md

```diff
@@ -188,6 +188,51 @@ Execution forbidden
 
 Finally, to confirm that the server will allow permitted operations, try running an operation from the client.
 
+## Configuration
+
+### Selective enforcement
+
+In some cases, deployments may want to selectively enable the behavior of `forbidUnregisteredOperations` depending on environmental conditions (e.g. based on headers).
+
+To selectively enable operation safe-listing, the `forbidUnregisteredOperations` setting supports a predicate function which receives the request context and can return `true` or `false` to indicate whether enforcement is enabled or disabled respectively.
+
+For example, to enforce the operation registry safe-listing while skipping enforcement for any request in which the `Let-me-pass` header was present with a value of `Pretty please?`, the following configuration could be used:
+
+```js line=12-27
+const server = new ApolloServer({
+  // Existing configuration
+  typeDefs,
+  resolvers,
+  subscriptions: false,
+  engine: "<ENGINE_API_KEY>",
+  plugins: [
+    require("apollo-server-plugin-operation-registry")({
+      // De-structure the object to get the HTTP `headers` and the GraphQL
+      // request `context`.  Additional validation is possible, but this
+      // function must be synchronous.  For more details, see the note below.
+      forbidUnregisteredOperations({
+        context, // Destructure the shared request `context`.
+        request: {
+          http: { headers } // Destructure the `headers` class.
+        }
+      }) {
+        // If a magic header is in place, allow any unregistered operation.
+        // The `headers` implements the Fetch API which is documented here:
```

Comment:
I would move the sentence that headers implements the Fetch API up to the main content since it's an important detail.

Reply:
Thank you, and done via https://github.com/apollographql/apollo/pull/333/commits/c00b07ae49635f71a9243c4d7acc30ff8108141c.